dbt_project.yml

```diff
@@ -1,10 +1,10 @@
 name: 'linkedin_source'
-version: '0.7.0'
+version: '0.8.0'
 config-version: 2
 require-dbt-version: [">=1.3.0", "<2.0.0"]
 models:
   linkedin_source:
-    +materialized: view
+    +materialized: table
```

Comment:
We should probably highlight this change in the CHANGELOG.

We should also have it match the format of our other source packages. See [here](https://github.com/fivetran/dbt_twitter_source/blob/1ab9900882dec2f367ead5b0be61431272269873/dbt_project.yml#L8-L12) for an example.

Reply:
I didn't mean to release this--I have reverted back to the original view.